specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/Builders.json

```diff
@@ -356,6 +356,14 @@
           "type": "string",
           "description": "Resource ID of the compute that the patching will be performed on.",
           "readOnly": true
+        },
+        "identityMappings": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IdentityMappings"
+          },
+          "x-ms-identifiers": [],
+          "description": "List of mappings of managed identities to the container registry that they allow access to."
```

Comment:
Hope I didn't forget about something. So at this point the relationship is 1-1 (one managed identity for one container registry), right?
Please consider updating the description to something like `List of mappings of a managed identity with the container registry that it allows access to. (1-1 relationship)`

Reply:
Hmm, you do raise a good point of confusion. So we allow for each container registry to have a single managed identity, however, the same managed identity could be used for multiple container registries. So actually, the best data structure for this relationship is going to be a dictionary of container registry -> identity mappings, with the registry login server being the key. I've updated to reflect that new structure